README.md

```diff
@@ -81,7 +106,12 @@ Type
 	make check
 
 
-## How to use ##
+## Optimization ##
+
+The default compilation flag is "-O3". One may use the "-march=native -O3" flag to optimize the binaries. See https://github.com/fplll/fplll/issues/169 for its impact on the enumeration speed.
```

Comment:
Nitpicking: Maybe `-O3` instead of "-O3". We could also do the [this issue](https://github.com/fplll/fplll/issues/169)